kubetest/e2e.go

```diff
@@ -587,11 +587,16 @@ func nodeTest(nodeArgs []string, testArgs, nodeTestArgs, project, zone string) e
 func kubemarkUp(dump string, o options, deploy deployer) error {
 	// Stop previously running kubemark cluster (if any).
 	if err := control.XMLWrap(&suite, "Kubemark TearDown Previous", func() error {
-		return control.FinishRunning(exec.Command("./test/kubemark/stop-kubemark.sh"))
+		if err := control.FinishRunning(exec.Command("./test/kubemark/stop-kubemark.sh")); err != nil {
+			return fmt.Errorf("failed to stop kubemark cluster, err: %v", err)
+		}
+		return nil
 	}); err != nil {
 		return err
 	}
 
+	fmt.Println("finished tearing down kubemark")
```

Comment:
I think that you should use log.Printf here instead of fmt.Println to be consistent with other print calls in this file.